persistent-postgresql/ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for persistent-postgresql
 
+## 2.11.1
+
+* Implement interval support. #1053
```

Comment:
```suggestion
* Implement interval support. [#1053](https://github.com/yesodweb/persistent/pull/1053)
```

This will get batched in with `persistent-2.11.0.0`, since that is not yet released. Could you move the bullet down there?

Reply:
Updated